install/sct_launcher

```diff
@@ -101,8 +101,14 @@ if [ -z "${SCT_LOADED_ONCE}" ]; then
 fi
 if [[ "${0}" == "${BASH_SOURCE}" ]] && [[ -n "${1}"  ]]; then
   echo "Running ${SCT_DIR}/scripts/${@}"
-   python ${SCT_DIR}/scripts/${@}
+  start=$(date +%s)
+
+  python ${SCT_DIR}/scripts/${@}
+
+  end=$(date +%s)
+  runtime=$(python -c "print '%u:%02u' % ((${end} - ${start})/60, (${end} - ${start})%60)")
```

Comment:
I would prefer using just bash to show that, otherwise you'll increase execution time of every `sct_*` call by starting an entire python interpreter to show elapsed time.

Reply:
@perone I'm pretty sure that launching an inline Python command such that would take at maximum only a few milliseconds, but I've changed it to calculate in bash, anyway.